torch/_inductor/lowering.py

```diff
@@ -2586,6 +2590,7 @@ def _local_scalar_dense(data):
     binding_sym, keypath = next(iter(unbacked_bindings.items()))
     buffer = ir.DynamicScalar(binding_sym, keypath, data)
     buffer.name = V.graph.register_buffer(buffer)
+    V.graph.register_operation(buffer)
```

Comment:
is a `DynamicScalar`both a buffer and an operation? That doesn't seem right?

Reply:
Yes I added the `OperationBuffer` class to make the transition easier. These objects are registered both as operations and as buffers.